_drgn.pyi

```diff
@@ -954,6 +954,9 @@ class Architecture(enum.Enum):
     RISCV32 = ...
     """The 32-bit RISC-V architecture."""
 
+    S390 = ...
+    """The s390 architecture, a.k.a. IBM Z."""
```

Comment:
Thanks for the explanation of s390 vs s390x. drgn does actually have some support for userspace debugging, so I think we need definitions for both 31-bit s390 and s390x.

Reply:
We have no plans to support 31-bit mode - s390/IBM Z has support for 64 bit since 2000. We dropped 31-bit support from the Linux kernel for quite some time now, the only thing we still have is CONFIG_COMPAT to let customers run old 31-bit binaries in userspace. But i'm not aware of anyone doing development on 31-bit. So 31-bit support would add a lot more complexity to the code with no addtional gain.

Reply:
I see. I was aware that the kernel doesn't support 31 bit s390, but I didn't realize it was that uncommon for userspace. I agree that it's not worth supporting it. However, it's not too much effort to at least detect 31-bit s390 and have a stub architecture definition for it -- that's what I did in my branch.